adam/ontology/phase1_ontology.py

```diff
@@ -2436,7 +2426,19 @@ def _make_jump_description() -> Iterable[Tuple[OntologyNode, ActionDescription]]
                 objects_to_paths=[
                     (jump_agent, SpatialPath(AWAY_FROM, JUMP_INITIAL_SUPPORTER_AUX)),
                     (jump_agent, SpatialPath(AWAY_FROM, jump_ground)),
-                ]
+                ],
+                # must be above the ground at some point during the action
+                at_some_point=[
+                    Relation(
+                        IN_REGION,
+                        jump_agent,
+                        Region(
+                            reference_object=JUMP_INITIAL_SUPPORTER_AUX,
+                            distance=DISTAL,
```

Comment:
Does someone jump 'DISTAL' from the ground? I would have thought you jump `PROXIMAL`

Reply:
Both of these are `at_some_point` relations because I wanted to specify that when you jump you leave the ground at some point -- i.e. you're on it then far away from it then on it again. Or is `PROXIMAL` a better choice because it indicates you don't go too far?

Reply:
I think `PROXIMAL` is a better choice here, but if we're consistent in choice it's not a huge deal. I just wasn't sure if there was previous conversation which led to the choice of `DISTAL` @gabbard do you have any opinion here?

Reply:
I think `THROW` uses `DISTAL`, which is part of why I chose it, but I get the argument for `PROXIMAL`. I can change it if you both think that's a good idea

Reply:
We can skip with Distal